src/node.cpp

```diff
@@ -1800,6 +1825,11 @@ bool LocalNode::serialize(string* d)
     d->append((char*)&ll, sizeof ll);
     d->append(localname.data(), ll);
 
+    const auto syncableInt = static_cast<int8_t>(!syncable);
```

Comment:
perhaps a formulation such as `char syncableInt = syncable ? 0 : 1;`  similar to lines above would be a lot clearer?
Also, on reanalysis of this and the corresponding deserializer,  this code serializing and then the older SDK code deserializing would break because the old code expects not just a flag, but a number of bytes to skip over.   It would read the new flag as 1, skip 1 byte further, and expect 5 more flags (actually the number of bytes to skip each time), but there would only be 4, and everything after would be misaligned.  thanks